nuget/Controls.nuspec

```diff
@@ -1,56 +1,56 @@
+     <file src="src\Controls\Controls.Android\bin\Release\XamarinCommunityToolkit.Controls.Android.xml" target="lib\MonoAndroid10" />
 
      <!--Xamarin.iOSUnified-->
-     <file src="src/Controls/Controls/bin/Release/FormsCommunityToolkit.Controls.dll" target="lib/Xamarin.iOS10/FormsCommunityToolkit.Controls.dll" />
-     <file src="src/Controls/Controls/bin/Release/FormsCommunityToolkit.Controls.pdb" target="lib/Xamarin.iOS10/FormsCommunityToolkit.Controls.pdb" />
-     <file src="src/Controls/Controls/bin/Release/FormsCommunityToolkit.Controls.xml" target="lib/Xamarin.iOS10/FormsCommunityToolkit.Controls.xml" />
-     <file src="src/Controls/Controls.iOS/bin/Release/FormsCommunityToolkit.Controls.iOS.dll" target="lib/Xamarin.iOS10/FormsCommunityToolkit.Controls.iOS.dll" />
-     <file src="src/Controls/Controls.iOS/bin/Release/FormsCommunityToolkit.Controls.iOS.pdb" target="lib/Xamarin.iOS10/FormsCommunityToolkit.Controls.iOS.pdb" />
-     <file src="src/Controls/Controls.iOS/bin/Release/FormsCommunityToolkit.Controls.iOS.xml" target="lib/Xamarin.iOS10/FormsCommunityToolkit.Controls.iOS.xml" />
-     <!--UWP-->
-     <file src="src/Controls/Controls/bin/Release/FormsCommunityToolkit.Controls.dll" target="lib/uap10.0/FormsCommunityToolkit.Controls.dll" />
-     <file src="src/Controls/Controls/bin/Release/FormsCommunityToolkit.Controls.pdb" target="lib/uap10.0/FormsCommunityToolkit.Controls.pdb" />
-     <file src="src/Controls/Controls/bin/Release/FormsCommunityToolkit.Controls.xml" target="lib/uap10.0/FormsCommunityToolkit.Controls.xml" />
-     <file src="src/Controls/Controls.UWP/bin/Release/FormsCommunityToolkit.Controls.UWP.dll" target="lib/uap10.0/FormsCommunityToolkit.Controls.UWP.dll" />
-     <file src="src/Controls/Controls.UWP/bin/Release/FormsCommunityToolkit.Controls.UWP.pdb" target="lib/uap10.0/FormsCommunityToolkit.Controls.UWP.pdb" />
-     <file src="src/Controls/Controls.UWP/bin/Release/FormsCommunityToolkit.Controls.UWP.xml" target="lib/uap10.0/FormsCommunityToolkit.Controls.UWP.xml" />
+     <file src="src\Controls\Controls\bin\Release\XamarinCommunityToolkit.Controls.dll" target="lib\Xamarin.iOS10" />
+     <file src="src\Controls\Controls\bin\Release\XamarinCommunityToolkit.Controls.pdb" target="lib\Xamarin.iOS10" />
+     <file src="src\Controls\Controls\bin\Release\XamarinCommunityToolkit.Controls.xml" target="lib\Xamarin.iOS10" />
+     <file src="src\Controls\Controls.iOS\bin\Release\XamarinCommunityToolkit.Controls.iOS.dll" target="lib\Xamarin.iOS10" />	 
+     <file src="src\Controls\Controls.iOS\bin\Release\XamarinCommunityToolkit.Controls.iOS.pdb" target="lib\Xamarin.iOS10" />
+     <file src="src\Controls\Controls.iOS\bin\Release\XamarinCommunityToolkit.Controls.iOS.xml" target="lib\Xamarin.iOS10" />
+    
+	<!--UWP-->
+     <file src="src\Controls\Controls\bin\Release\XamarinCommunityToolkit.Controls.dll" target="lib\uap10.0" />
```

Comment:
Can be simplified to src\Controls\Controls\bin\Release\XamarinCommunityToolkit.Controls.* requiring only 1 line instead of 3? On the other hand, explicit file names gives us the error in case e.g. xml document file is missing.

Reply:
I have reviewed and tested it. I have simplified every nuspec.
![imagen](https://user-images.githubusercontent.com/6755973/35995914-2654d864-0d15-11e8-8508-c58a083fbbc2.png)